packages/grafana-ui/src/components/Table/TableNG/styles.ts

```diff
@@ -145,6 +147,21 @@ export const getDefaultCellStyles: TableCellStyles = (theme, { textAlign, should
     },
   });
 
+export const getMaxHeightCellStyles: TableCellStyles = (_theme, { textAlign, maxHeight }) =>
+  css({
+    display: 'flex',
+    alignItems: 'center',
+    textAlign,
+    justifyContent: getJustifyContent(textAlign),
```

Comment:
we essentially have to reapply the thinnest layer of the outer styles again on the inner container if it exists for maxHeight, which I also dislike. But it was much less complex to do this than the thing I was originally doing which either put the styles on the rdg-cell or this height container, that just wasn't working when it came to the overflow sizing.